drivers/serial/uart_nrfx_uart.c

```diff
@@ -395,6 +410,15 @@ static int uart_nrfx_init(struct device *dev)
 
 #ifdef CONFIG_UART_INTERRUPT_DRIVEN
 
+	/* Global variable: m_init_uart_event_txrdy is necessary to make
+	 * API functions: uart_nrfx_irq_tx_complete and uart_nrfx_irq_tx_ready
+	 * return always correct value.
+	 * These API functions will return:
```

Comment:
Could we just, rephrase, the following lines, to be more "to the point" ? E.g.
"The aforementioned API functions will always return true before the driver has transmitted any characters, regardless of the status of the TX_READY event flag. After that the functions will always return NRF_UART_EVENT_RXDRDY after the driver has transmitted the first character".

Reply:
Sure I can use it.